src/Sylius/Bundle/ApiBundle/Resources/config/api_platform/resources/Customer.xml

```diff
@@ -66,10 +168,37 @@
                 </openapiContext>
             </operation>
 
-            <operation
-                class="ApiPlatform\Metadata\Get"
-                uriTemplate="/shop/customers/{id}"
-            >
+            <operation class="ApiPlatform\Metadata\Put" uriTemplate="/shop/customers/{id}" >
```

Comment:
The order of operations is `get`, `post`, `put`, but now we have `get` at the end.